backend/src/main/resources/db/migration/schema/V003__Projekt_uebersicht_indexes.sql

```
@@ -0,0 +1,3 @@
+create index idx_projekt__projektnummer_id on projekt (projektnummer, id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)